hive/stackable/bin/start-metastore

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 #
+# This file was created by the Stackable developers.
```

Comment:
Suggestion to remove copy/pasted content
```
# This file was created by the Stackable developers.
#
# Usage: start-metastore <options>
# 
# Checks if the metastore database schema is initialized. If so it starts the metastore,
# otherwise it tries to initialize the schma first.
#
```

Reply:
I don't understand

Reply:
The usage description contains namenode etc. 
```
# Usage: start-metastore <options> namenode [namenode-options]
```

Reply:
I see. Done.